pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -61,8 +67,23 @@ class LightningGPUAdjointJacobian(LightningBaseAdjointJacobian):
 
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, qubit_state: LightningGPUStateVector, batch_obs: bool = False) -> None:
+    def __init__(
+        self,
+        qubit_state: LightningGPUStateVector,
+        batch_obs: bool = False,
+        use_mpi: bool = False,
+        mpi_handler=None,
+    ) -> None:
+
         super().__init__(qubit_state, batch_obs)
+
+        self._dp = DevPool()
+
+        self._use_mpi = use_mpi
+
+        if use_mpi:
```

Comment:
Usually prefer class attrs over copied args:
```suggestion
        if self._use_mpi:
```